identity-iota-core/src/types/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+pub use network::Network;
+pub use network::NetworkName;
+
+mod network;
```

Comment:
Nit: consider moving `network.rs` from `types` to `did`. Still undecided on this since the `MessageIdNull` implementation would not have a natural home apart from `types`...

Reply:
Moved `MessageIdExt` and `Network` to the `tangle` sub-module.